src/NHSD.BuyingCatalogue.Solutions.Persistence/Repositories/PriceRepository.cs

```diff
@@ -13,23 +13,23 @@ public sealed class PriceRepository : IPriceRepository
 
         private const string ListCataloguePricesSql = @"
         SELECT  cp.CatalogueItemId,
-                ci.[Name] as CatalogueItemName,
+                ci.[Name] AS CatalogueItemName,
                 cp.CataloguePriceId,
                 cp.CataloguePriceTypeId,
                 pu.PricingUnitId,
-                pu.[Name] as PricingUnitName,
-                pu.[Description] as PricingUnitDescription,
-                pu.TierName as PricingUnitTierName,
+                pu.[Name] AS PricingUnitName,
+                pu.[Description] AS PricingUnitDescription,
+                pu.TierName AS PricingUnitTierName,
                 cp.TimeUnitId,
                 cp.CurrencyCode,
                 cp.Price AS FlatPrice,
                 cptr.BandStart,
                 cptr.BandEnd,
                 cptr.Price AS TieredPrice
         FROM    CataloguePrice cp
-                INNER JOIN PricingUnit pu ON pu.PricingUnitId = cp.PricingUnitId
-                INNER JOIN CatalogueItem ci ON ci.CatalogueItemId = cp.CatalogueItemId
-                LEFT OUTER JOIN CataloguePriceTier cptr ON cptr.CataloguePriceId = cp.CataloguePriceId
+                INNER JOIN dbo.PricingUnit pu ON pu.PricingUnitId = cp.PricingUnitId
+                INNER JOIN dbo.CatalogueItem ci ON ci.CatalogueItemId = cp.CatalogueItemId
+                LEFT OUTER JOIN dbo.CataloguePriceTier cptr ON cptr.CataloguePriceId = cp.CataloguePriceId
```

Comment:
I'm not sure if it was one of the comments that disappeared but you just need to add the `AS` keyword to your table aliases (and the schema qualifier, `dbo`, to `CataloguePrice`), e.g.:

```sql
FROM dbo.CataloguePrice AS cp
     INNER JOIN dbo.PricingUnit AS pu
```

Reply:
Ahhh I get you, it was there, I think I just misunderstood 😄